README.md

```diff
@@ -27,6 +27,36 @@ Installation instructions for the CLI
 
 This repository does not need any specific configuration.
 
+
+### Docker
+
+To build a local image for Zonemaster CLI you need a [local Zonemaster Engine
+base image].
+
+Build a new local base image:
+
+```sh
+make docker-build
+```
+
+Tag the local base image with the current version number:
+
+```sh
+make docker-tag-version
+```
+
+Tag the local base image as the latest version:
+
+```sh
+make docker-tag-latest
+```
+
+Test a zone using the local base image:
+
+```sh
+sudo docker run zonemaster/cli:local zonemaster.net
+```
```

Comment:
To remove the container after it has run and stopped:
```sh
sudo docker run --rm zonemaster/cli:local zonemaster.net
```

Reply:
How do you detect it's still running? When I run this I get an empty listing:

```sh
sudo docker run zonemaster/cli:local zonemaster.net ; sudo docker ps
```

Reply:
```
sudo docker ps -a
```
lists stopped containers.

Reply:
Fixed